recipes/graphene/all/conanfile.py

```diff
@@ -31,14 +31,17 @@ class LibnameConan(ConanFile):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
+        if self.settings.compiler == "gcc":
+            if tools.Version(self.settings.compiler.version) < "5.0":
+                raise ConanInvalidConfiguration("graphene does not support GCC before 5.0")
```

Comment:
Was it pure luck that the previous version compiled with 4.9?

Reply:
I think it's caused by this change in glib: https://github.com/GNOME/glib/commit/c1d74e35c18aae1ffb2959e4e8c9d1e8c60103ba#diff-c918d2aee000c84eef78e92218bf0ca515b50777bff2fa991b267bb2fbfcf3de
The error is:
```
/home/conan/w/cci_PR-4586/.conan/data/glib/2.67.1/_/_/package/cfa5e581e5fc6cde76da25751420fe10555ca7e5/include/glib-2.0/glib/gmacros.h:953:15: error: "_MSC_VER" is not defined [-Werror=undef]
 #elif 1200 <= _MSC_VER
               ^
/home/conan/w/cci_PR-4586/.conan/data/glib/2.67.1/_/_/package/cfa5e581e5fc6cde76da25751420fe10555ca7e5/include/glib-2.0/glib/gmacros.h:957:8: error: "__cplusplus" is not defined [-Werror=undef]
 #elif (__cplusplus >= 201103 && !(__GNUC__ == 4 && __GNUC_MINOR__ == 7)) || (_MSC_VER >= 1900)
        ^
/home/conan/w/cci_PR-4586/.conan/data/glib/2.67.1/_/_/package/cfa5e581e5fc6cde76da25751420fe10555ca7e5/include/glib-2.0/glib/gmacros.h:957:78: error: "_MSC_VER" is not defined [-Werror=undef]
 #elif (__cplusplus >= 201103 && !(__GNUC__ == 4 && __GNUC_MINOR__ == 7)) || (_MSC_VER >= 1900)
```
It does not happen when building glib package because glib does not use `-Werror=undef` contrary to graphene